src/main/java/uk/gov/companieshouse/api/accounts/interceptor/TransactionInterceptor.java

```diff
@@ -34,6 +34,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             String transactionId = pathVariables.get("transactionId");
             ResponseEntity<Transaction> transaction = transactionManager
                     .getTransaction(transactionId, request.getHeader("X-Request-Id"));
+            request.setAttribute("transaction", transaction);
```

Comment:
This PR places the transaction in request scope but https://github.com/companieshouse/company-accounts.api.ch.gov.uk/pull/18 is placing the transaction in the session scope. Can you agree a consistent approach please?